cypress/integration/cucumber/step_definitions/refapp-3.x/05-vitals-and-triage/vitals-and-triage.js

```diff
@@ -0,0 +1,103 @@
+import { Given, Before, When } from "cypress-cucumber-preprocessor/steps";
+
+let identifier = null;
+let patient = null;
+
+Before({ tags: "@vitals-and-triage" }, () => {
+  cy.generateIdentifier().then((generatedIdentifier) => {
+    identifier = generatedIdentifier;
+    cy.createPatient(identifier).then((generatedPatient) => {
+      patient = generatedPatient;
+      cy.startFacilityVisit(patient.uuid);
+    });
+  });
+});
+
+Given("the user is logged in", () => {
+  cy.on("uncaught:exception", (err, runnable) => {
+    console.log(err);
+    return false;
+  });
+  cy.login();
+});
+
+Given("the user arrives on a patient’s summary page", () => {
+  cy.visit(`patient/${patient.uuid}/chart`);
+});
+
+When("the user clicks on Edit patient details", () => {
+  cy.contains("Actions").click({ force: true });
+  cy.contains("Edit patient details").click({ force: true });
+});
+
+Then("the edit page should load", () => {
+  cy.contains("Edit Patient");
+});
+
+When("the user updates the address", () => {
+  cy.get("#address1").clear({ force: true }).type("45 baker's street");
+  cy.get("#country").clear({ force: true }).type("Germany");
+  cy.get("#stateProvince").clear({ force: true }).type("Central");
+  cy.get("#cityVillage").clear({ force: true }).type("Kandy");
+  cy.get("#postalCode").clear({ force: true }).type("3000");
+  cy.contains("Update Patient").click({ force: true });
+});
+
+Then("the address should be updated", () => {
+  cy.contains("Update Patient").click({ force: true });
+});
+
+When("the user clicks on Record Vitals", () => {
+  cy.contains("Record Vitals").click({ force: true });
+});
+
+Then("the Vitals form should load", () => {
+  cy.contains("Record Vitals and Biometrics");
+});
+
+When("the user adds vitals", () => {
+  cy.get("#systolic").type("120", { force: true });
```

Comment:
Do not select items by Ids or CSS selectors unless there's a specific reason. 

You can use `cy.getByLabel("myLabel")` to get input by its label. 

https://github.com/openmrs/openmrs-contrib-qaframework/blob/d23bebab1def3cfd2351e2afa1ed4d307a35b92b/cypress/support/commands.js#L31

Reply:
what if we use id for now and create an issue in MF to change this. Then change this.

Reply:
Okay let's do it. Also add a todo comment.